tests/log/test_log_level.py

```diff
@@ -0,0 +1,53 @@
+from logging import DEBUG, getLogger
+
+from inspect_ai import Task, eval
+from inspect_ai._util.constants import SANDBOX
+from inspect_ai.log._log import EvalLog
+from inspect_ai.log._transcript import LoggerEvent
+from inspect_ai.model import get_model
+from inspect_ai.solver import Generate, TaskState, solver
+
+logger = getLogger(__name__)
+
+
+def test_log_file_level() -> None:
+    @solver
+    def logging_solver(level_no: int):
+        async def solve(state: TaskState, generate: Generate):
+            logger.log(level_no, "sandbox log entry")
+            return state
+
+        return solve
+
+    def find_logger_event(log: EvalLog) -> LoggerEvent | None:
+        if log.samples:
+            return next(
+                (
+                    event
+                    for event in log.samples[0].transcript.events
+                    if isinstance(event, LoggerEvent)
+                ),
+                None,
+            )
+        else:
+            return None
+
+    log = eval(
+        Task(solver=logging_solver(SANDBOX)),
+        model=get_model("mockllm/model"),
+        log_level="debug",
+        log_file_level="sandbox",
+    )[0]
+
+    event = find_logger_event(log)
+    assert event
+    assert event.message.level == "sandbox"
+
+    log = eval(
+        Task(solver=logging_solver(DEBUG)),
+        model=get_model("mockllm/model"),
+        log_level="debug",
+        log_file_level="warning",
```

Comment:
I find "file" a bit confusing for the distinction between the log output destinations, especially reading this unit test: the log_file_level determine the logs in the EvalLog object(s), which (in my mind anyway) isn't necessarily a file

Reply:
Yeah, I didn't love that either but struggled to come up with an alternative. We could do `--log-transcript-level` ?